src/main/java/com/altenheim/kalender/controller/logicController/ImportController.java

```diff
@@ -1,8 +1,38 @@
 package com.altenheim.kalender.controller.logicController;
 
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.text.ParseException;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.ZoneId;
+import java.util.GregorianCalendar;
+
 import com.altenheim.kalender.interfaces.ICalendarEntriesModel;
 import com.altenheim.kalender.interfaces.IImportController;
 import com.altenheim.kalender.interfaces.IWebsiteScraperController;
+import com.calendarfx.model.Entry;
+
+import net.fortuna.ical4j.data.CalendarBuilder;
+import net.fortuna.ical4j.data.CalendarOutputter;
+import net.fortuna.ical4j.data.ParserException;
+import net.fortuna.ical4j.model.Calendar;
+import net.fortuna.ical4j.model.ComponentList;
+import net.fortuna.ical4j.model.DateTime;
```

Comment:
Guckt euch mal in der Lib das Modelspackage an. Wenn das jetzt nur unswesentlich mehr sind als die, die ihr importiert (ich sag mal so bis max, 5) könnt ihr das mit Wildcard machen.  Oben sind leere Zeilen.

Reply:
Sind zu viel (deutlich mehr als 5 mehr)